src/txdb.h

```diff
@@ -74,6 +74,8 @@ class CCoinsViewDB final : public CCoinsView
 class CCoinsViewDBCursor: public CCoinsViewCursor
 {
 public:
+    CCoinsViewDBCursor(CDBIterator* pcursorIn, const uint256 &hashBlockIn):
```

Comment:
Is there a reason to put this into the header? Previously the constructor was private, thus disallowing "illegal" access. Now, with it being public, the whole class could be moved to the cpp file?

Reply:
Good call! No reason this needs to be exposed; moved to the .cpp file.